app.js

```diff
@@ -1,9 +1,13 @@
+const bodyParser = require('body-parser');
 const express = require('express');
 
 const feedRoutes = require('./routes/feed');
 
 const app = express();
 
+// app.use(bodyParser.urlencoded()); //  x-www-form-urlencoded Use with form <form>
```

Comment:
We don't need `app.use(bodyParser.urlencoded());` because we didn't using UI form But we're using `app.use(bodyParser.json());` to read the request body in a json format